### ISSUE TITLE
[UX] - Function Monitor not displaying any data - App Insights half working

### ISSUE THREAD

**Issue by flatproject** [Azure/Azure-Functions#1651]
# Issue Title
My function Monitor is not displaying any invocation traces.

## Description
I created a new Timer Function (runs every 5 min) from within Visual Studio and deployed it to Azure as a package on a Consumption plan. Created a new Application Insights and connected it to my app via the Azure Function configuration settings (just added the appi key).
My function runs some .Net core 3.1.5 code on a timer on a Consumption plan.

- Please provide a concise description of the issue
   The Azure Monitor screen is not displaying any invocations. I know my function works because it sends some emails every day.
 Also App Insights seems to show some sort of data on the Overview page but when going to logs or Performance or Availability or Failures shows nothing ( i know my app was failing for two days as i was trying to understand the issue) 

- Include Screenshots if possible
![InvocationScreen](https://user-images.githubusercontent.com/18282373/87028884-30dada80-c1d7-11ea-9194-3cd8ed5ec548.PNG)
![Overview](https://user-images.githubusercontent.com/18282373/87028888-31737100-c1d7-11ea-8508-df564718bf5e.PNG)
![AppInsightsOverview](https://user-images.githubusercontent.com/18282373/87028890-31737100-c1d7-11ea-9cf7-19d813393f98.PNG)
![Configuration](https://user-images.githubusercontent.com/18282373/87028891-32a49e00-c1d7-11ea-8fc1-16d368a1f909.PNG)
![hostson](https://user-images.githubusercontent.com/18282373/87028948-48b25e80-c1d7-11ea-9ce1-1e345b8ec504.PNG)
![localsettings](https://user-images.githubusercontent.com/18282373/87028950-494af500-c1d7-11ea-8f65-9af620b54aa0.png)

What is wrong? I have recreated the same function 5 times up to now and cannot get the Monitor to show any invocations.

I have about 7 other functions that work perfectly fine. I have compared the configurations and all look the same.

Any ideas what else to try?

Thanks

**Comment by btardif** [comment]
@ehamai can you help with this one?

**Comment by ehamai** [comment]
@brettsam and @ammanifold for awareness

Can you try the classic Functions management experience to see if our old logging mechanism is working for you?  Some customers have been seeing issues with our newer App Insights log streaming so we're planning on bringing it back soon, but for now, it would be helpful to see if you have any logs at all.  Based on the fact that you're not seeing anything in your invocation logs makes me think that something else is going on.

**To go back to the classic experience**
1. Go to the Overview blade for your function app
2. On the top right, there is a button that says "Switch to classic experience"
3. Navigate to your function code editor and open logs at bottom

**Also if you can provide us with a sessionId of your repro, we can also check if there's any obvious runtime issues on our side:**
1. Reproduce the issue
2. Open your browser dev console (Usually F12)
3. Click on the console tab
4. Copy the sessionId from the console logs

**Comment by flatproject** [comment]
Thanks @ehamai 

This is the log output from the classic experience Logs:
`2020-07-09T16:10:00.006 [Information] Executing 'AutoXXXXXXX' (Reason='Timer fired at 2020-07-09T17:10:00.0055959+01:00', Id=af808b07-ef09-4f67-9356-f65f6169d5ed)
2020-07-09T16:10:01.832 [Information] Executed 'AutoXXXXXXX' (Succeeded, Id=af808b07-ef09-4f67-9356-f65f6169d5ed)
2020-07-09T16:11:07  No new trace in the past 1 min(s).`

Its worth noting that the Logs (next to Invocations) works fine as well.

session: `cea1ff367e194c4da8dc6f55edcf52dd`

![Untitled](https://user-images.githubusercontent.com/18282373/87064979-56340c80-c208-11ea-8ab8-876603890ec8.png)

Also clicking on "Run query in Application Insights"

![Capture](https://user-images.githubusercontent.com/18282373/87065377-dd818000-c208-11ea-8b7c-10eeabb0c500.PNG)

doesn't show any results

![Capture2](https://user-images.githubusercontent.com/18282373/87065428-eb370580-c208-11ea-9988-dbd06162a553.PNG)

**Comment by flatproject** [comment]
The Azure portal was giving me the option to create App Insights in UKWest region when I raised this ticket.

Went on holidays for 3 weeks came back and all my App Insight UKWest resources had vanished. I thought probably I had accidentally deleted them. 

Created a new App Insight again in the UKWest region and I noticed that the new resource wasn't giving me a KEY to use in my apps/func. At that point I had enough trying to get this working so I left it for another week.

A week later I tried to find this new App Insight resource and I couldn't find it so I thought I'll give it a last chance. Went to create a new App Insight resource and this time I didn't have the UKWest option so I created it in the UKSouth region.

Everything works now!


Ticket can be closed!
